Source/ImPlotWidgets/Private/HistogramWidget.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "ImPlotWidgets/Public/HistogramWidget.h"
+
+#include <ImGui/Public/ImPlotImplementation.h>
+#include <Runtime/Core/Public/Algo/Sort.h>
+
+void SHistogramWidget::Construct(const FArguments& InArgs)
+{
+	SImPlotWidget::Construct(InArgs);
+}
+
+void SHistogramWidget::OnImGuiDraw() const
+{
+	SImPlotWidget::OnImGuiDraw();
+
+	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
+
+	constexpr int32 GraphFlags = ImPlotFlags_AntiAliased;
+	constexpr int32 AxisFlags = ImPlotAxisFlags_NoGridLines;
+
+    // ---------------------------------
+    {
+		static int bins = 50;
+		ImGui::SetNextItemWidth(AllottedGeometry.GetLocalSize().X);
+		if (bins >= 0) 
+		{
+			ImGui::SameLine();
+			ImGui::SetNextItemWidth(AllottedGeometry.GetLocalSize().X);
+			ImGui::SliderInt("##Bins", &bins, 1, 100);
+		}
+
+        float rmin = FMath::Min(SImPlotWidget::InData);;
+        float rmax = FMath::Max(SImPlotWidget::InData);
+
+		ImPlot::SetNextPlotLimits(rmin, rmax, 0, rmax * (rmax * 0.2f / bins), ImGuiCond_Always);
```

Comment:
What is the 0.2f for here, can we promote it to a variable to give more context?